tests/MongoDB.Driver.Tests/UpdateDefinitionBuilderTests.cs

```diff
@@ -238,6 +238,18 @@ public void Inc_Typed()
             Assert(subject.Inc("Age", 1), "{$inc: {age: 1}}");
         }
 
+        [Fact]
+        public void Indexed_Positional_Typed()
+        {
+            var subject = CreateSubject<Person>();
+
+#pragma warning disable
```

Comment:
Can we just disable the necessary warning instead of all?

Reply:
Done